src/Covid19Dashboard/overlays/GoogleMobilityLayer/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as ReactMapGL from 'react-map-gl';
+
+/*
+  Use Mapbox filters to selection all counties not in Illinois and render them
```

Comment:
is this comment accurate?

Reply:
no longer. leftover from the template example from Craig